ts/features/bonus/bpd/analytics/index.ts

```diff
@@ -63,6 +63,11 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
       return mp.track(action.type, { reason: action.payload.message });
 
     // transactions
+    case getType(bpdTransactionsLoad.failure):
+      return mp.track(action.type, {
+        awardPeriodId: action.payload,
```

Comment:
This shouldn't be:
```suggestion
        awardPeriodId: action.payload.awardPeriodId,
```
?

Reply:
Yes it should 😄